roachtest: bump PredecessorVersion(20.2) to 20.1.5

arulajmani opened the pull request:
Release justification: change part of 20.1.5 release process
Release note: None

arulajmani committed:
roachtest: bump PredecessorVersion(20.2) to 20.1.5

Release justification: change part of 20.1.5 release process
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/53668)
<!-- Reviewable:end -->

jlinder commented on pkg/cmd/roachtest/test_runner.go:
```diff
@@ -1158,7 +1158,7 @@ func PredecessorVersion(buildVersion version.Version) (string, error) {
 	// (see runVersionUpgrade). The same is true for adding a new key to this
```
Does this comment need to be followed to update the fixtures (as was done [for these releases](https://github.com/cockroachdb/cockroach/pull/50770))?

arulajmani replied:
Yeah, I think so. Seems like checkpointing the 20.1.5 fixtures requires the binaries to be downloadable from the homepage -- so I'm gonna wait on this PR until that's done.

arulajmani committed:
roachtest: update stale checkpointing invocation message

The directory structure of logs is slightly different now that we have
redacted logs, so the checkpoints are created in a different location.
Update the (on purpose) error message to reference the correct location.

Release note: None

arulajmani commented:
@jlinder sorry for the delay on closing the loop on this, I've updated the checkpointing error message as well so that we don't run into this problem again.

jlinder reviewed (approved):
LGTM

arulajmani commented:
bors r=jlinder

arulajmani commented:
closing this in favour of #54761

arulajmani commented:
closing this in favour of #54761
